applications/mpsky/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: "mpsky"
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "mpsky.selectorLabels" . | nindent 6 }}
+  policyTypes:
+    - "Ingress"
+  ingress:
+    # Allow inbound access from pods (in any namespace) labeled
+    # gafaelfawr.lsst.io/ingress: true.
+    - from:
+        - namespaceSelector: {}
+          podSelector:
+            matchLabels:
+              gafaelfawr.lsst.io/ingress: "true"
+      ports:
+        - protocol: "TCP"
+          port: 8080
```

Comment:
Is the NetworkPolicy here necessary, given that gafaelfawr and ingress are out of the picture?